redash/query_runner/athena.py

```diff
@@ -125,18 +125,20 @@ def __get_schema_from_glue(self):
                 region_name=self.configuration['region']
                 )
         schema = {}
-        paginator = client.get_paginator('get_tables')
+        database_paginator = client.get_paginator('get_databases')
+        table_paginator = client.get_paginator('get_tables')
+        databases = (database for databases in database_paginator.paginate()
+                     for database in databases['DatabaseList'])
```

Comment:
something in these 2 lines doesn't look right. are you sure this is what you wanted?

Reply:
We want to paginate over both databases and tables.
Database paginator return list of lists, in the l#130 I flatten it inside a generator.

Reply:
Maybe it would suffice to just indent the double for-loops differently to make it more clear?